config/input-resource-verify.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project name="input-validate" default="execute">
```

Comment:
Any folders this config scans, it should pickup all files in them.
Examining cachefile generated shows what files it is finding.